packages/scxa-tsne-plot/src/TSnePlotView.js

```diff
@@ -119,26 +128,47 @@ class TSnePlotView extends React.Component {
 
     return (
       <div className={wrapperClassName}>
+        <div className={`row expanded`}>
+          <div className={`small-12 medium-6 columns`}>
+            <PlotSettingsDropdown
+                labelText={`Plot type:`}
+                options={plotTypesOptions}
+                defaultValue={{value: selectedPlotType, label: plot.plotType}}
+                onSelect={(selectedPlotType) => {
+                  onChangePlotTypes(selectedPlotType.value)
+                }}/>
+          </div>
+          <div className={`small-12 medium-6 columns`}>
+            <PlotSettingsDropdown
+                labelText={plotOptionsLabel}
+                options={plotOptionsValues}
+                defaultValue={{value: selectedParameter,
+                  label: selectedPlotType == `tSNE` ? `perplexity=` + selectedParameter : `n_neighbor=` + selectedParameter}}
```

Comment:
Please replace `==` with `===` as `==` is a loose equality operator in javascript.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness